snappass/main.py

```diff
@@ -162,6 +162,7 @@ def index():
 @app.route('/', methods=['POST'])
 def handle_password():
     ttl, password = clean_input()
+    json = request.form.get('json',False,type=bool)
```

Comment:
```suggestion
    json = request.form.get('json', False, type=bool)
```

Reply:
Its probably better to switch on the accept header which flask already parses for us: https://flask.palletsprojects.com/en/1.0.x/api/#flask.Request.accept_mimetypes

Reply:
Per the suggestion, you can drop this line and below do:

```py
if request.accept_mimetypes.accept_json:
    return jsonify(link=link, ttl=ttl)

return render_template('confirm.html', password_link=link)
```